### PR TITLE
feat: Rename flow states

### PR DESCRIPTION
**Pull request by dharmendrak**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-1435

### Change description ###

feat: Rename flow states

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -147,6 +147,54 @@
...
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "CLAIM_ISSUE",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[APPLICANTSOLICITORONE]",
+          "[CREATOR]",
```
Comment (ckalista) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json]: Why do we need [CREATOR] case role here? By then the case role should have been removed.

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -147,6 +147,54 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "PAYMENT_FAILED",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[APPLICANTSOLICITORONE]",
+          "[CREATOR]",
```
Comment (ckalista) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json]: Why do we need [CREATOR] case role here? By then the case role should have been removed.

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 3]
```diff
@@ -147,6 +147,54 @@
...
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseEventID": "PROCEEDS_IN_HERITAGE_SYSTEM",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[APPLICANTSOLICITORONE]",
+          "[CREATOR]",
```
Comment (ckalista) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json]: Why do we need [CREATOR] case role here? By then the case role should have been removed.

File: ccd-definition/CaseEvent/Camunda/CaseEvents.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "ShowEventNotes": "N",
+    "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "PAYMENT_FAILED",
+    "Name": "Payment failed",
+    "Description": "Payment failed",
+    "PreConditionState(s)": "*",
```
Comment (ckalista) [inline ccd-definition/CaseEvent/Camunda/CaseEvents.json]: We should be stricter here: pre condition state `PRENDING_CASE_ISSUED` same as post condition state

**Review by ckalista — DISMISSED** [review]
Just a few comments 👍

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json [hunk 5]
```diff
@@ -147,6 +147,51 @@
       }
     ]
   },
+  {
```
Comment (swalker125) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json:150]: auth for system update user should really be defined here now to have one source of truth

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json [hunk 6]
```diff
@@ -1717,6 +1717,26 @@
       }
     ]
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "CaseFieldID": "takenOfflineByStaffDate",
```
Comment (swalker125) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:1723]: why is this needed?

File: ccd-definition/CaseEvent/Camunda/CaseEvents.json [hunk 7]
```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "CLAIM_ISSUE",
+    "Name": "Issue claim",
+    "Description": "Issue claim",
+    "PreConditionState(s)": "*",
```
Comment (swalker125) [inline ccd-definition/CaseEvent/Camunda/CaseEvents.json]: why any predcondition state?

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json (hunk 5 above)
Comment (dharmendrak) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json:150]: done

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json (hunk 6 above)
Comment (dharmendrak) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json:1723]: needed to separate the taking offline by user/system

**Review by sabahirfan — APPROVED** [review]

**Review by ckalista — APPROVED** [review]
